slashing-protection/src/main/resources/migrations/postgresql/V2__removeUniqueConstraints.sql

```diff
@@ -0,0 +1,19 @@
+DO
+$body$
+DECLARE
+  _signed_block_conn text := (
+  SELECT conname
+  FROM pg_constraint
+  WHERE conrelid = 'signed_blocks'::regclass
+  AND contype = 'u');
+
+  _signed_attestation_conn text := (
+  SELECT conname
+  FROM pg_constraint
+  WHERE conrelid = 'signed_attestations'::regclass
+  AND contype = 'u');
+BEGIN
+  EXECUTE 'ALTER TABLE signed_blocks DROP CONSTRAINT ' || _signed_block_conn;
```

Comment:
Does removing these constraints affect performance at all? Essentially this was acting a primary key before